ios/RCCTabBarController.m

```diff
@@ -166,6 +166,10 @@ - (instancetype)initWithProps:(NSDictionary *)props children:(NSArray *)children
     id selectedIcon = tabItemLayout[@"props"][@"selectedIcon"];
     if (selectedIcon) {
       iconImageSelected = [RCTConvert UIImage:selectedIcon];
+      
+      if (!selectedButtonColor) {
```

Comment:
I can't set an original icon for selected state without UIImageRenderingModeAlwaysOriginal. Without this condition, selectedIcon are always colored with the default (light-blue) color.